src/core/inlines.js

```diff
@@ -109,3 +123,58 @@ export function run(conf) {
     txt.parentNode.replaceChild(df, txt);
   }
 }
+
+function generateIDLMarkup(ref) {
+  const { base, attribute, method, args } = parseInlineIDL(ref);
+
+  if (!method && !attribute) {
+    return hyperHTML`<code><a class="respec-idl-xref">${base}</a></code>`;
+  }
+
+  const code = hyperHTML`<code></code>`;
+
+  if (base) {
+    code.appendChild(hyperHTML`<a class="respec-idl-xref">${base}</a>`);
+    code.appendChild(document.createTextNode("."));
```

Comment:
doesn't: 
```
code.appendChild(hyperHTML`<a class="respec-idl-xref">${base}</a>.`);
```

Work?

Reply:
No it doesn't :disappointed: 
`"Failed to execute 'appendChild' on 'Node': parameter 1 is not of type 'Node'."` as there are two nodes here

Reply:
Ah, right. I see what’s going on. On my phone now, but I think all this can be collapsed into the `<code>` above.

Reply:
I've done some edits. Will update :)